force-app/main/default/staticresources/lmsvf/lmsvf.js

```diff
@@ -0,0 +1,149 @@
+"use strict";
+
+var lmsutil = window.lmsutil; 
```

Comment:
Please do a global search/replace so that we have this in camelCase everywhere: `lmsUtil`

Reply:
Just to be clear, you're concerned with the var name? I'd rather not have to rename the static resource.

Reply:
We should probably rename both so that it's more readable and in line with the rest of our JS code.

Reply:
Actually, wires crossed. This isn't a static resource reference.